### ISSUE TITLE
Can you configure authentication (-auth-file) with an environment variable?

### ISSUE THREAD

**Issue by JamesValleyT** [mailhog/MailHog#181]
https://github.com/mailhog/MailHog/blob/master/docs/Auth.md says you can pass " -auth-file=docs/example-auth" with a password file.

https://github.com/mailhog/MailHog/blob/master/docs/CONFIG.md talks about configuration with  command line options or environment variables but makes no mention of -auth-file 

Can you configure authentication (-auth-file) with an environment variable?

Thanks,
James

**Comment by andrew-jones** [comment]
Yes you can, its defined here https://github.com/mailhog/MailHog/blob/d6585029d60980c67bd988a18532f2b91d10f385/config/config.go#L34

**Comment by booooza** [comment]
IMHO it would be even better if you could configure basic auth directly via env variables, so you don't need to provide a password file. E.g. `MH_AUTH_USERNAME` and `MH_AUTH_PASSWORD` :)
